index.bs

```diff
@@ -642,10 +642,15 @@ ReadableStream(<var>underlyingSource</var> = {}, { <var>size</var>, <var>highWat
        * While WritableStreamDefaultWriterGetDesiredSize(_writer_) is ≤ *0* or is *null*, the user agent must not read
          from _reader_.
        * If _reader_ is a <a>BYOB reader</a>, WritableStreamDefaultWriterGetDesiredSize(_writer_) should be used to
-         determine the size of the chunks read from _reader_.
-       * Otherwise, WritableStreamDefaultWriterGetDesiredSize(_writer_) may be used to determine the flow rate
-         heuristically, e.g. by delaying reads while it is judged to be "low" compared to the size of chunks that have
-         been typically read.
+         determine the size of the chunks read from _reader_. Otherwise, the desired size may be used to determine the
+         flow rate heuristically, e.g. by delaying reads while the desired size is judged to be "low" compared to the
+         size of <a>chunks</a> that have been typically read.
+       * After taking into account these backpressure signals, reading and writing should be done as fast as possible;
```

Comment:
"as possible" seems ambiguous to me. It could have a wide variety of meanings from "you must optimise this at the machine code level for your target architecture" to "someone said we could maximise battery life by only forwarding data once a minute, so that's the fastest possible".

"promptly" feels right to me. "reading and writing should be done promptly". Not necessarily instantly, but not wasting time.